cmd/dispatch-daemon/main.go

```diff
@@ -21,8 +22,15 @@ func init() {
 
 func main() {
 	config.InitLogger(config.DevMode)
+	logger := config.InitLogger(config.Mode(os.Getenv("MODE")))
 
-	from := "daha.kyiv@gmail.com"
+	from := os.Getenv("MAILMAL_EMAIL")
```

Comment:
What do you think about place config file near to main where you will provide all of the config variables and make validation. Check this out https://github.com/caarlos0/env

Reply:
You can do this also in near services

Reply:
Great job!
Give you some advices about ENV variables in new round of review